package.json

```diff
@@ -9,7 +9,7 @@
 		"native",
 		"debug"
 	],
-	"version": "0.24.0",
+	"version": "0.25.0",
```

Comment:
please don't change the version property (I change it on release when I call vsce and make it a separate commit)

Reply:
Reverted that, sorry.